homeassistant/components/config/config_entries.py

```diff
@@ -280,8 +266,25 @@ async def system_options_update(hass, connection, msg):
     if entry is None:
         return
 
+    old_disable_polling = entry.system_options.disable_polling
```

Comment:
Should we make a full copy here so we are ready for the next entry to be added to it?

Reply:
We don't know if the next system option is going to require a reload.